packages/components/src/popover/index.js

```diff
@@ -137,25 +137,26 @@ const Popover = (
 		: placement;
 
 	const ownerDocument = useMemo( () => {
+		let documentToReturn;
+
 		if ( anchorRef?.top ) {
```

Comment:
I'm surprised `anchorRef` is not even documented in the readme, when it's the prop that takes the highest priority. Looks like we have a lot of work to do (docs, stories, tests) as we work out the kinks in this component 💪

Reply:
Yup! This component definitely needs a lot of work